.env.example

```diff
@@ -19,3 +19,10 @@ MYSQL_PORT=3306
 REDIS_HOST=redis
 
 STRIPE_API_VERSION=2022-08-01
+
+# Generic client timeout
+CLIENT_TIMEOUT=15
+
+# Mailer vars
+MAILER_API=https://mailer-staging.thebiggivetest.org.uk
+MAILER_SEND_SECRET=
```

Comment:
@alihejazi GitHub flags this because (outside of Salesforce world) it's usually normal practice to include a trailing newline, again primarily for cleaner diffs